res/mixxx.qrc

```diff
@@ -13,6 +13,7 @@
         <file>images/library/ic_library_banshee.svg</file>
         <file>images/library/ic_library_computer.svg</file> <!-- from Gnome high contrast icons LGPL 2.1 -->
         <file>images/library/ic_library_crates.svg</file>
+        <file>images/library/ic_library_clementine.png</file>
```

Comment:
Can you add a svg file instead?

The clementine logo is here: https://en.wikipedia.org/wiki/File:Clementine_logo.svg

Reply:
could someone help me on this? i am not a inkscape magician :/

Reply:
Maybe @ronso0?

Reply:
@ronso0 can you help me on this minijob on creating a logo=?

Reply:
Let's keep this *unresolved* until fixed as suggested.

Reply:
resolved by adding a svg